tape/src/main/java/com/squareup/tape/QueueFile.java

```diff
@@ -523,25 +523,34 @@ public synchronized void remove() throws IOException {
     if (elementCount == 1) {
       clear();
     } else {
-      // assert elementCount > 1
       int firstTotalLength = Element.HEADER_LENGTH + first.length;
 
-      ringErase(first.position, firstTotalLength);
-
+      // Read the next element
```

Comment:
nit: period. [Comments should be sentences](nedbatchelder.com/blog/201401/comments_should_be_sentences.html).